gdc_client/parcel/segment.py

```diff
@@ -94,10 +94,11 @@ def validate_segment_md5sums(self, path=None):
 
         log.debug("Checksumming {0}:".format(self.download.url))
 
-        pbar = tqdm(iterable=intervals, file=sys.stdout)
+        pbar = get_percentage_pbar(len(intervals))
 
         with mmap_open(path or self.download.path) as data:
-            for interval in pbar:
+            for i, interval in enumerate(intervals):
+                pbar.update(i + 1)
```

Comment:
I think this is a little different where we used to update with the size_complete? and it was later in the flow? https://github.com/NCI-GDC/gdc-client/pull/262/files#diff-8d934cd5147a38b2e9a623cdc45d3259L284 I could be reading this wrong though.

Reply:
this is a different progressbar. it's for md5sum validation, the one you're pointing to is for file download